tests/test_data.py

```diff
@@ -112,6 +115,51 @@ def test_terminate_backend(postgresql, data):
 
 def test_pg_get_active_connections(data, execute):
     assert data.pg_get_active_connections() == 1
-    execute("select pg_sleep(1)")
-    time.sleep(1)
-    assert data.pg_get_active_connections() == 2
+    execute("select pg_sleep(2)")
+    nbconn = wait_for_data(
+        data.pg_get_active_connections,
+        msg="could not get active connections",
+    )
+    assert nbconn == 2
+
+
+def test_encoding(postgresql, data, execute):
+    """Test for issue #149"""
+    postgresql.set_session(autocommit=True)
+    with postgresql.cursor() as cur:
+        """plateform specific locales (,Centos, Ubuntu)"""
+        for encoding in ["fr_FR.latin1", "fr_FR.88591", "fr_FR.8859-1"]:
+            try:
+                cur.execute(
+                    f"CREATE DATABASE latin1 ENCODING 'latin1' TEMPLATE template0 LC_COLLATE '{encoding}' LC_CTYPE '{encoding}'"
+                )
+            except WrongObjectType:
+                continue
+            else:
+                break
+
+    postgresql.set_session(autocommit=False)
+    execute("CREATE TABLE tbl(s text)", dbname="latin1", commit=True)
+    execute(
+        "INSERT INTO tbl(s) VALUES ('initilialized éléphant')",
+        dbname="latin1",
+        commit=True,
+    )
+    execute("UPDATE tbl SET s = 'blocking éléphant'", dbname="latin1")
+    execute("UPDATE tbl SET s = 'waiting éléphant'", dbname="latin1", commit=True)
+    running = wait_for_data(data.pg_get_activities, msg="could not fetch activities")
+    assert "blocking éléphant" in running[0].query
+    (waiting,) = data.pg_get_waiting()
+    assert "éléphant" in waiting.query
+    (blocking,) = data.pg_get_blocking()
+    assert "éléphant" in blocking.query
```

Comment:
`waiting éléphant` and `blocking éléphant` here as well :)

Reply:
right https://github.com/dalibo/pg_activity/pull/212/commits/4ec554aa74e463ed04d0b9c3f95347d3036b1b0a

Reply:
thx